2-mandatory/1-create-functions.js

```diff
@@ -3,15 +3,17 @@ Write a function that:
 - Accepts an array as a parameter.
 - Returns a new array containing the first five elements of the passed array.
 */
-function first5() {
-}
+function first5(arr) {
+  return arr.slice(0,5);
+ }
 
 /*
 Write a function that:
 - Accepts an array as a parameter.
 - Returns a new array containing the same elements, except sorted.
 */
-function sortArray() {
+function sortArray(arr) {
+  return arr.slice().sort();
```

Comment:
Something to be wary of is that the javascript `.sort()` assumes things are strings, so it will sort `[2,3,1,11, 23]` as [1,11,2,23,3]`. For this reason, it's best to pass a comparision function in.  https://www.w3schools.com/js/js_array_sort.asp